src/vivarium_public_health/population/add_new_birth_cohorts.py

```diff
@@ -205,8 +205,8 @@ def setup(self, builder: Builder) -> None:
 
     def load_age_specific_fertility_rate_data(self, builder: Builder) -> pd.DataFrame:
         asfr_data = builder.data.load("covariate.age_specific_fertility_rate.estimate")
-        columns = ["year_start", "year_end", "age_start", "age_end", "mean_value"]
-        asfr_data = asfr_data.loc[asfr_data.sex == "Female"][columns]
+        columns = ["year_start", "year_end", "age_start", "age_end", "value"]
+        asfr_data = asfr_data.loc[(asfr_data.sex == "Female") & (asfr_data.parameter == "mean_value")].reset_index()[columns]
```

Comment:
Is it necessary to subset to the second condition here?

Reply:
I don't remember!